src/algebra/algebra/basic.lean

```diff
@@ -1267,7 +1269,7 @@ def to_rat_alg_hom [ring R] [ring S] [algebra ℚ R] [algebra ℚ S] (f : R →+
 
 end ring_hom
 
-namespace rat
+section rat
```

Comment:
Do you mean for these to be in `_root_` now?

Reply:
Yes, to match `algebra_nat` and `algebra_int`. The `algebra_map_rat_rat` lemma seems fine in the global namespace too, rather than mentioning `rat` three times when the statement only has it twice as we do now!